internal/nodes/avi_vrf_translator.go

```diff
@@ -89,6 +93,20 @@ func (o *AviObjectGraph) addRouteForNode(node *v1.Node, vrfName string, routeid
 			break
 		}
 	}
+	if ipFamily == v6Type {
+		if lib.GetCNIPlugin() == lib.CALICO_CNI {
+			if ip, ok := node.Annotations["projectcalico.org/IPv6Address"]; ok {
+				nodeIP6 = strings.Split(ip, "/")[0]
+			}
+			if nodeIP6 == "" {
+				utils.AviLog.Errorf("Error in fetching nodeIPv6 for %v", node.ObjectMeta.Name)
```

Comment:
Why are we erroring out here?  Do we want to have route table have only ipv6 route        
 -->next hop IPV6 and IPV4 route --> next hop ipv4

Reply:
yes, that is the expectation